### PR TITLE
CNF Certification results for v1.0-beta/matrixx-digital-commerce

### PR DESCRIPTION
**Pull request by EricLo-417**
## Pre-submission checklist:

Please check each of these after submitting your pull request:

- [x]  If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/cnf-certification/blob/main/Certified_CNF_Form.md)?
 - [x] Did you include the product/project logo in SVG, EPS or AI format?
 - [x] Does your logo clearly state the name of the product/project/org and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
 - [ ] If your product/project is open source, did you include the repo_url?
 - [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Comment by taylor** [comment]
@EricLo-417 a few things:

- please change the folder to v1.0-beta1 instead of v0.29.0 (certification vs test suite)
- please test with the [v0.29.1 release](https://github.com/cncf/cnf-testsuite/releases/tag/v0.29.1) and submit results

**Comment by agentpoyo** [comment]
@EricLo-417 ,

Looks like the summary posted states `results/cnf-testsuite-results-20220515-170901-190.yml` was generated but looks like that results didn't get updated if you ran v0.29.1 in the files.

**Review by taylor — CHANGES_REQUESTED** [review]
@EricLo-417 @agentpoyo I see the following in the README
```
2022-05-15 21:33:58 : [runTest001]    [cnfRun]          - 25 of 56 total tests passed
2022-05-15 21:33:58 : [runTest001]    [cnfRun]          - 10 of 13 essential tests passed
2022-05-15 21:33:58 : [runTest001]    [cnfRun]        Results have been saved to results/cnf-testsuite-results-20220515-170901-190.yml
```

note the 0515 vs 0517 for the results.

The results file is v1.0-beta/matrixx-digital-commerce/cnf-testsuite-results-20220517-173143-843.yml

**Comment by SomethingCreativeStudios** [comment]
Oh I am sorry that was just supposed to be an example of the output. I figured the results file was the thing to matter. Do you still want this sample output updated?

**Comment by agentpoyo** [comment]
> @EricLo-417 @agentpoyo I see the following in the README
> 
> ```
> 2022-05-15 21:33:58 : [runTest001]    [cnfRun]          - 25 of 56 total tests passed
> 2022-05-15 21:33:58 : [runTest001]    [cnfRun]          - 10 of 13 essential tests passed
> 2022-05-15 21:33:58 : [runTest001]    [cnfRun]        Results have been saved to results/cnf-testsuite-results-20220515-170901-190.yml
> ```
> 
> note the 0515 vs 0517 for the results.
> 
> The results file is v1.0-beta/matrixx-digital-commerce/cnf-testsuite-results-20220517-173143-843.yml

True, I was looking at the last 3 numbers in comparison but I noticed the results file attached to the PR can't be from v0.29.1 since it's missing the tags for the type of test (essential, normal, bonus) and the command that was used to run. @taylor @EricLo-417

**Comment by EricLo-417** [comment]
Updated results to use version 0.29.1
Updated readme's example output to use latest results

File: v1.0-beta/matrixx-digital-commerce/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+vendor: MATRIXX Software
+name: Matrixx Digital Commerce
+version: 5241
+website_url: www.matrixx.com
+product_logo_url: https://www.dropbox.com/s/ywhib7dh6fxysjl/matrixx_logo_software_color_cmyk.eps?dl=0
```
Comment (taylor) [inline v1.0-beta/matrixx-digital-commerce/PRODUCT.yaml:5]:
```suggestion
product_logo_url: https://www.dropbox.com/s/wrpx3vv1rkowg3h/matrixx_logo_software_color_cmyk.svg?dl=0
```

**Review by taylor — CHANGES_REQUESTED** [review]
Switch to SVG version

File: v1.0-beta/matrixx-digital-commerce/PRODUCT.yaml (hunk 1 above)
Comment (EricLo-417) [reply v1.0-beta/matrixx-digital-commerce/PRODUCT.yaml]: Updated

**Review by taylor — APPROVED** [review]
lgtm

**Review by denverwilliams — APPROVED** [review]

**Comment by denverwilliams** [comment]
lgtm

**Comment by lixuna** [comment]
Remaining Tasks to Complete After PR Review

- [x] Update the Certified CNF spreadsheet to reflect the vendor's certified offering.
- [ ] (in progress) Add the vendor's information to the CNCF landscape which also causes them to appear on https://www.cncf.io/certification/cnf.
- [ ] Add a comment saying "This product is now a "Certified CNF" and merge the PR.

**Comment by taylor** [comment]
This product is now a "Certified CNF"

**Pull request merged**